src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -121,6 +133,10 @@ public function handleException($exception, $request, $response)
      */
     protected function getRenderer($exception)
     {
+        if (!$this->renderer) {
+            $this->renderer = $this->config('exceptionRender') ?: ExceptionRenderer::class;
+        }
```

Comment:
Any reason this couldn't be done in the constructor? Are you trying to avoid the class load?